src/main/java/frc/robot/subsystems/BallPath/Shooter/ShooterImpl.java

```diff
@@ -38,11 +108,253 @@ public int checkBalls(){
     }
 
     @Override
-    public void setHoodAngle(double distance){}
+    public void setHoodAngle(double setPointHood){
+        if(turretHoodPosition >= setPointHood - 20000 && turretHoodPosition <= setPointHood + 20000){
```

Comment:
Please extract this `20000` to a constant value so that if we need to update it, it's only one reference that needs to be changed.